observatorium/overlays/moc/smaug/thanos/horizontalpodautoscalers/observatorium-thanos-receive-default.yaml

```diff
@@ -0,0 +1,19 @@
+---
+kind: HorizontalPodAutoscaler
+apiVersion: autoscaling/v2beta2
+metadata:
+  name: observatorium-thanos-receive-default
+spec:
+  scaleTargetRef:
+    kind: StatefulSet
+    name: observatorium-thanos-receive-default
+    apiVersion: apps/v1
+  minReplicas: 5
+  maxReplicas: 25
```

Comment:
this value seems really high, this would result in ~200 cores and 800gb of memory at capacity...which is like 20% of smaug's cpu and 13% of its memory. Can we reduce this?

Reply:
updated to 10 `maxReplicas`